files/default/handler/audit_report.rb

```diff
@@ -209,6 +209,13 @@ def gather_nodeinfo
           environment: n.environment,
           roles: runlist_roles,
           recipes: runlist_recipes,
+          policy_name: node.policy_name || '',
```

Comment:
why are some of these `node.blabla` and some `n.blabla` ?

Reply:
No good reason, `node` and `n` are the same. I pushed a new commits to use `n` in that function. Thank you